samples/java-valueentity-counter-spring-client/src/main/java/com/example/client/spring/rest/controller/CounterController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.client.spring.rest.controller;
+
+import com.example.client.spring.rest.model.CounterRequest;
+import com.example.client.spring.rest.service.CounterService;
+import com.example.client.spring.rest.service.GrpcClientService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+@RestController
+public class CounterController {
+
+  @Autowired
+  CounterService counterService;
+
+  @Autowired
+  GrpcClientService grpcClientService;
+
+  @PostMapping(path = "/getCurrentCounter")
```

Comment:
we could make it more RESTful. 

For instance, this method could be a GET on `/counter` with the id in the path, like `GET /counter/my-counter-id

Reply:
Yes that makes sene, I will change them. Currently I just kept them same as of Akka Serverless API

Reply:
Updated it.